source/lex.tex

```diff
@@ -1330,7 +1472,7 @@
 
 \begin{bnf}
 \nontermdef{conditional-escape-sequence-char}\br
-    \textnormal{any member of the basic character set that is not an} octal-digit\textnormal{, a} simple-escape-sequence-char\textnormal{, or the characters \terminal{u}, \terminal{U}, or \terminal{x}}
+\textnormal{any member of the basic character set that is not an} octal-digit\textnormal{, a} simple-escape-sequence-char\textnormal{, or the characters \terminal{N}, \terminal{u}, \terminal{U}, or \terminal{x}}
```

Comment:
We want the spaces in front; please restore them.

Reply:
This was an accident, and I'll revert the spacing change, but it does seem that the bnf environment laid it out the same without the spaces? Preserving the code readability is enough reason, of course.